openmdao/recorders/tests/test_sqlite_recorder.py

```diff
@@ -1635,7 +1635,7 @@ def assert_closed(self, recorder):
         prob.setup()
 
         driver = prob.driver
-        system = prob.model.pz
+        system = prob.model._auto_ivc
         solver = prob.model.nonlinear_solver.linesearch = om.BoundsEnforceLS()
```

Comment:
I don't think we want to teach people to record on a system by showing them a hidden system they've never seen before.  Sadly, this test needs to be rewritten to have a recorder added on some other component.

Reply:
I changed it to a different component, but I don't think this is something that shows up in the docs, it's just a test that was broken by the auto_ivc updates to sellar.py.